src/components/Capture/ProfileData.tsx

```diff
@@ -265,9 +265,20 @@ const getTranslatedFieldHelperText = (
 ) => {
   const specificTranslation = translateSpecific('helper_text', type, country)
 
-  return specificTranslation ? (
-    <HelperText>{translate(`profile_data.${specificTranslation}`)}</HelperText>
-  ) : null
+  if (specificTranslation) {
+    return (
+      <HelperText>
+        {translate(`profile_data.${specificTranslation}`)}
+      </HelperText>
+    )
+  }
+
+  switch (type) {
+    case 'dob':
+      return <HelperText>MM / DD / YYYY</HelperText>
```

Comment:
Why do we change the labels for non US but keep the dates as American format for all languages?